homeassistant/components/airzone_cloud/sensor.py

```diff
@@ -64,13 +64,15 @@
 ZONE_SENSOR_TYPES: Final[tuple[SensorEntityDescription, ...]] = (
     SensorEntityDescription(
         device_class=SensorDeviceClass.TEMPERATURE,
+        has_entity_name=True,
         key=AZD_TEMP,
         name="Temperature",
```

Comment:
This is now the default when `has_entity_name` is set and a device class is provided.

```suggestion
```

Reply:
Removed in 7c203a693eca064c81edfcc39e7f44bc58e04f7e